features/support/conjur_helper.rb

```diff
@@ -36,6 +36,12 @@ def store_secret_in_remote_conjur(var_id, value)
     end
   end
 
+  def get_secret_from_remote_conjur(var_id)
```

Comment:
```suggestion
  def remote_conjur_secret(var_id)
```

Reply:
I was modeling this after the above `store_secret_in_remote_conjur` but yeah this is fine. Will change.